src/hooks/submit-form.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
```

Comment:
I have mentioned having the custom_hook into a new commit so it will be easier.

Please split this into two, have the addition of this with the calls of this in project-registration and student-registration

Reply:
If you could also mention what are the advantages of using that then it would be good in the commit message.